views/layouts/layout.njk

```diff
@@ -24,6 +32,7 @@
       {% include "_footer.njk" %}
       </div>
     </div>
+    <script src="/javascripts/application.js"></script>
+    <script src="/javascripts/components/mobile-navigation.js"></script>
```

Comment:
These should be concatenated into `application.js` in `lib/metalsmith.js` – like [this](https://github.com/alphagov/govuk-design-system/blob/273e71c5b5cfeab6b65b563f4cf666319ca4b6fe/lib/metalsmith.js#L52-L63).

Annoyingly I _think_ Modernizr needs to be included in the `<head>` in order to shim HTML5 elements, so we might not be able to move this to the bottom of the body without addressing that.